include/boost/math/tools/random_vector.hpp

```diff
@@ -25,12 +25,12 @@ std::vector<T> generate_random_vector(std::size_t size, std::size_t seed)
     }
     std::vector<T> v(size);
 
-    std::mt19937 gen(seed);
+    std::mt19937_64 gen(seed);
 
     std::normal_distribution<T> dis(0, 1);
-    for(std::size_t i = 0; i < v.size(); ++i)
+    for (auto& i : v)
```

Comment:
I might use `auto& x : v` as we generally will assume `dis(gen)` produces a floating point number.

Reply:
Done!